addons/website_version/models/experiment.py

```diff
@@ -0,0 +1,300 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
```

Comment:
Well here I would reorder the code
 - models/version.py -> holds various version-related models
 - models/google_management.py -> google management class
 - models/experiment.py -> not necessary (why separate website_version.version

About naming convention: except for config_parameter that is a mistake, we separate names in model name by dots. This means that you should have website_version.experiment.version, not experiment_version. This convention is used in the whole codebase, so I think we should use it here. It is very annoying when suddenly, some piece of code uses another convention and that you have to recheck everything carefully.